apps/wallet-mobile/src/features/Swap/useCases/ConfirmTxScreen/ConfirmTxScreen.tsx

```diff
@@ -134,7 +137,7 @@ export const ConfirmTxScreen = () => {
               secureTextEntry
               ref={spendingPasswordRef}
               enablesReturnKeyAutomatically
-              label={strings.spendingPassword}
+              placeholder={strings.spendingPassword}
```

Comment:
@SorinC6 This was complaining (label doesn't exist). Could you check what would be the best option?